functions-and-scope.js

```diff
@@ -53,12 +95,30 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
 
+console.log("\nOpdracht 2b:")
+
+function averageGrade(array) {
+    let sum = 0;
+    for (let i = 0; i < array.length; i++) {
+        sum += array[i];
+    }
+    const average = sum / array.length;
+    return average;
+}
+
+console.log(averageGrade(grades));
+console.log(averageGrade([6, 4, 5]));
+console.log(averageGrade([8, 9, 4, 6, 10]));
+
 
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
 
-
+console.log("\nOpdracht 2c:")
+console.log((Math.round(averageGrade(grades) * 100) / 100));
```

Comment:
Op zich werkt dit, maar je zou hier de toFixed functie voor kunnen gebruiken:
```suggestion
console.log(averageGrade(grades).toFixed(2));
```